src/external/bot-skeleton/scratch/backward-compatibility.js

```diff
@@ -527,7 +520,7 @@ export default class BlockConversion {
 
         this.workspace = null;
 
-        window.window.window.Blockly.Events.enable();
+        Blockly.Events.enable();
```

Comment:
@rupato-deriv can we use `window.Blockly` instead?

Reply:
done ✅